lib/iex/lib/iex/config.ex

```diff
@@ -157,7 +157,7 @@ defmodule IEx.Config do
   end
 
   def handle_configure(tab, options) do
-    options = :lists.ukeysort(1, options)
+    options = List.keysort(options, 0) |> Enum.uniq_by(fn {k, _} -> k end)
```

Comment:
I tried to look for a way to do this in only one pass, but I wasn't able to find it, which leads me to think if we need to add a function to the API, or extend the existing one (which is kind of difficult becuase we already have List.keysort/2 and /3) in case we want to add an `:options` field.
If there is interest, I can think of something and write a proposal.